src/components/DeviceMemoryHealth.tsx

```diff
@@ -0,0 +1,70 @@
+import StandartLineChart from './StandardLineChart'
+import ShadowBox from './ShadowBox'
+import IconText from './IconText'
+import { Paragraph, Separator, XStack, YStack } from 'tamagui'
+
+type DataPoint = {
+  x: number
+  y: number
+}
+
+type ChartData = {
+  id: string
+  color: string
+  data: DataPoint[]
+}
+
+type DeviceMemoryProps = {
+  load: number[]
```

Comment:
Here we probably don't want to call it "load" because load is a CPU thing. We also want the `maxMemory` or `availableMemory` to be passed as a single argument, plus something like `usedMemory` as an array of data points of the memory usage.